components/list/list-item.js

```diff
@@ -195,30 +203,55 @@ class ListItem extends LinkMixin(RtlMixin(LitElement)) {
 		fastdom.measure(() => this.resizedCallback(this.offsetWidth));
 	}
 
+	get checked() {
+		return this._checked;
+	}
+
+	set checked(value) {
+		const oldVal = this.checked;
+		this._checked = value;
+		this.requestUpdate('checked', oldVal).then(() => {
+			const checkBox = this.shadowRoot.querySelector('.d2l-list-item-checkbox');
+			checkBox && (checkBox.checked = value);
+
+			if (typeof this.ref === 'undefined') {
+				this.ref = ListItem.uniqueRef();
+			}
+		});
+	}
+
+	get ref() {
```

Comment:
Seems like this custom setter/getter should not be necessary.

Reply:
This is now useless! 

- [ ] Clean up ref getter and setter